lib/graphlient/errors/error.rb

```diff
@@ -1,13 +1,6 @@
 module Graphlient
   module Errors
     class Error < StandardError
-      attr_reader :inner_exception
```

Comment:
Is there any `Error` raised anywhere without an `inner_exception`? If not, I would undo this and make sure to pass it through, but otherwise this change is fine.

Reply:
You could always use [the `exception#cause` method](http://www.virtuouscode.com/2013/12/25/exception-causes-in-ruby-2-1/) to get the cause. While it's implicit, It is the way if the user of this gem wants to know if there's any exception raised eariler. I would leave it as it is and send a separate PR to deprecate it for compatibility.

Reply:
You could always use [the `exception#cause` method](http://www.virtuouscode.com/2013/12/25/exception-causes-in-ruby-2-1/) to get the cause. While it's implicit, It is the way if the user of this gem wants to know if there's any exception raised eariler. I would leave it as it is and send a separate PR to deprecate it for compatibility.

Reply:
Nice! I haven't been using that, it seems to exist since Ruby 2.1.

Reply:
Nice! didn't know about that, I agree with switching to it in a separate PR 🙌